lib/logic/controller.js

```diff
@@ -191,7 +191,7 @@ Controller.prototype.squashMergeWithoutRewrite = function(pr, commitMessage) {
             pr.getDest().getRef(),
             commitMessage
         ))
-        .delay(1000)
+        .delay(this.githubDelays["POST_MERGE"])
```

Comment:
Too much Python? `this.githubDelays.POST_MERGE` would be clearer, and throughout.